### PR TITLE
CMR Documentation Button - Phase 2

### PR DESCRIPTION
**Pull request by tiffwong**
## Description
Updating the rest of the usage of the DocumentationButton to the CMR version

## Type of Change
- Non breaking change ('update', 'change')

## Note to Reviewers
~Wait for [https://github.com/linode/manager/pull/7137](https://github.com/linode/manager/pull/7137) to be merged first~

### CODE REVIEW

**Review by WilkinsKa1 — CHANGES_REQUESTED** [review]
Looking good- there are a few more places the new docs button needs to be added:

• Stackscript details
• LKE cluster details (replace old docs button with new)
• Domain details
• NodeBalancer details
• Longview details (replace old with new

**Review by WilkinsKa1 — APPROVED** [review]

**Review by johnwcallahan — COMMENTED** [review]
I vote for removing the "external" icon on hover; it's a bit jarring that it appears out of the main content area.

Screen Shot 2020-11-30 at 12 16 05 PM

On Domain Details, when the domain name is long and the buttons wrap, there should be some left padding:

Screen Shot 2020-11-30 at 12 20 34 PM

Also there's a regression present here (not on develop) in non-CMR:
Screen Shot 2020-11-30 at 12 25 19 PM

File: packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodeTable.test.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ describe('NodeTable', () => {
       await findByText('Ready');
     });
   });
-  it('includes the Pool ID', () => {
+  it.skip('includes the Pool ID', () => {
```
Comment (DevDW) [inline packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodeTable.test.tsx:27]: Do we still need to skip this? It passed when I ran the test without skipping it

**Review by DevDW — APPROVED** [review]
LGTM 👌 

What are your thoughts on a bit of right padding for the Docs button in the mobile view so it isn't flush with the page edge?

Screen Shot 2020-11-30 at 3 28 26 PM

File: packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodeTable.test.tsx (hunk 1 above)
Comment (tiffwong) [reply packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodeTable.test.tsx]: It failed when I originally tried to commit but it seems to run fine now so I removed the skip, thanks for checking this!

**Pull request merged**